public/sprites/underwater.json

```diff
@@ -5,43 +5,43 @@
 
     "tiles": [
         {
-            "name": "ground",
+            "style": "ground",
```

Comment:
This file is a sprite sheet. It still defines tiles by name. I think the mixed terminology is a bit tricky to reason about too. 

I think of it as a database. In a particular table the primary id can be called just id because the full id name is implicated by the table name.

When referencing the id from another table thought it is required to prefix it.

Users table has `id`, `name`
Email table has `id`, `email`
Account table must have `user_id`, and `email_id` to reference them.

Reply:
Fixed.

Reply:
Fixed.